Atomic/atomic.py

```diff
@@ -645,36 +645,62 @@ def gen_cmd(self, cargs):
             args.append(c)
         return " ".join(args)
 
+    def get_fq_name(self, image_info):
+        if len(image_info['RepoTags']) > 1:
+            raise ValueError("\n{} is tagged with multiple repositories. "
+                             "Please use a repository name instead.\n".format(self.image))
+        else:
```

Comment:
great, but:

sudo ./atomic info rsyslog

do you already have the rsyslog image from redhat in your host? what happens if you docker rmi the image and run  the command again? how do you gather the additional registries configured in docker? (for remote inspection ofc)